roles/ara_frontend_nginx/tasks/main.yaml

```diff
@@ -38,6 +38,20 @@
 
     - when: ara_api_fqdn is defined
       block:
+        # TODO: htpasswd has moved to community.general
+        # https://github.com/ansible/ansible/blob/7450e87615b0d9b1503a0f966bd3327901decb75/lib/ansible/config/ansible_builtin_runtime.yml#L3195-L3196
+        - name: Configure users in .htpasswd for authentication
+          htpasswd:
+            path: "{{ ara_api_external_auth_file }}"
+            username: "{{ item.username }}"
+            password: "{{ item.password }}"
+            owner: ara_nginx_user
```

Comment:
Don't these need {{ }} ? :-)

Reply:
good catch, I wrote this too late last night :p